code/modules/surgery/internal.dm

```diff
@@ -35,6 +44,10 @@
 		playsound(get_turf(organ), 'sound/effects/squelch1.ogg', 50, 1)
 
 /datum/surgery_step/insert_item/fail_step(mob/living/user, obj/item/organ/external/organ, obj/item/tool)
+	if(istype(tool, /obj/item/gripper/surgery))
+		var/obj/item/gripper/surgery/SG = tool
+		if(SG.wrapped)
+			tool = SG.wrapped
 	user.visible_message(
 		SPAN_WARNING("[user]'s hand slips, hitting [organ.get_surgery_name()] with \the [tool]!"),
```

Comment:
You may need to edit these messages to fit in with robots, I don't see how a robots hand would slip. But this can be done in a later PR.

Reply:
Done.
![image](https://user-images.githubusercontent.com/65828539/129371722-409e779c-ff24-4fe9-baea-2285be49cd03.png)